packages/contracts/controller/src/account.cairo

```diff
@@ -488,7 +422,7 @@ mod CartridgeAccount {
 
             let owner_signature = SignerSignature::Starknet(
                 (
-                    StarknetSigner { pubkey: self._owner.read().try_into().expect('zero-pubkey') },
+                    self.multiple_owners.get_stark_owner().expect('no-stark-owner'),
```

Comment:
If that's the case I'm happy :)). The argent implementation allows for "legacy" signatures. So instead of the signature being a deserialized `SignerSignature` enum it also allows for it to be a "legacy" stark signature of two felts. 
I wanted to keep this possibility here as well.
If we don't need it we can safely remove it, and I will be happy to do so, since it adds additional complexity.

Reply:
Yes let's remove it. In our case, we don't need backward compatibility with the old signers, so should be fine to remove